WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -1410,17 +1391,17 @@ title_lawspeaker = {
 		OR = {
 			religion = norse_pagan
 			religion = norse_pagan_reformed
+			religion = germanic_pagan
+			religion = germanic_pagan_reformed
 		}
 	}
 	
 	allowed_to_grant = {
 		OR = {
-			is_feudal = yes
-			is_tribal = yes
-		}
-		OR = {
```

Comment:
You removed the "OR" condition too, and if you remove feudal and tribal, it won't make sense to have that title.